package/lighthouse-crd.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: multiclusterservices.lighthouse.submariner.io
+spec:
+  group: lighthouse.submariner.io
+  version: v1
+  names:
+        kind: MultiClusterService
+        plural: multiclusterservices
+        singular: multiclusterservice
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required: ["clusterServiceInfo"]
+          properties:
+            clusterServiceInfo:
+              properties:
+                clusterID:
+                  type: "string"
+                  minimum: 1
+                clusterDomain:
+                  type: "string"
+                  minimum: 1
```

Comment:
I'm not clear on what `clusterDomain` is for.  You're forcing it to be specified but it is not used in code, at least yet. Perhaps we should remove this restriction until we have a use case for `clusterDomain`?